test/users.spec.ts

```diff
@@ -199,6 +199,35 @@ describe('Users', function () {
     });
   });
 
+  /**
+   * update a credential label for a user
+   */
+  it('update a credential label for a user', async () => {
+    const userId = currentUser.id;
+    const result = await kcAdminClient.users.getCredentials({
+      id: userId!,
+    });
+
+    expect(result.map(c => c.type)).to.include('password');
+
+    const credential = result[0];
+
+    kcAdminClient.setConfig({
+      requestConfig: {headers: {'Content-Type': 'text/plain'}},
```

Comment:
Why would we have to set this header here? The request method itself should be responsible for setting the correct headers.

Reply:
I'm trying to set it just for this special test case. I think we are globally setting the content-type to application/json but I'm trying to override it. Would you know a better way to do this @jonkoops ? Thanks!

Reply:
Pushed a commit that should resolve this issue